README.md

```diff
@@ -1356,6 +1357,87 @@ isNeg * (1-isNeg) = 0
 1. [Hexens Audit Report](https://github.com/Hexens/Smart-Contract-Review-Public-Reports/blob/main/Hexens_Polygon_zkEVM_PUBLIC_27.02.23.pdf)
 2. [Fix Commit](https://github.com/0xPolygonHermez/zkevm-rom/commit/2ddeffbed7c022e04032e6d56ed6c6fb14cc38dc#diff-353b5f6c54dee2e069c391d2e2e6e3f503853e1d20126225f13a4d2d70a0d445)
 
+## <a name="Zendoo-polynomial">24. Zendoo: Missing Polynomial Normalization after Arithmetic Operations</a>
+
+**Summary**
+
+Related Vulnerabilities: Bad Polynomial Implementation
+
+Identified By: [NCC Group](https://www.nccgroup.com/)
+
+Incorrect polynomial representation
+resulting from arithmetic operations may break as sumptions and lead to erroneous computations or
```

Comment:
Typo: Change "as sumptions" to "assumptions"

Reply:
Thank you! I have corrected this spelling error, please check.

Reply:
LGTM, thanks!